README.md

```diff
@@ -33,6 +33,10 @@
 
 This is a Podcast hosting summit website, These website feature pro podcast speakers, interactive sessions with the speakers, networking opportunities, and the chance to register for an exhibition worldwide.
 
+Original design idea by Cindy Shin in Behance.
```

Comment:
- [OPTIONAL] Good job so far by adding the acknowledgment section to your README but I think it can be better if you add also the link to the design 😉 🙃